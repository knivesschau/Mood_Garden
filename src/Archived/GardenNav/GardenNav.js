import React, {Component} from 'react';
import {Link} from 'react-router-dom';
import TokenService from '../../services/token-services';
import './GardenNav.css'

export default class RoseNav extends Component {
    handleLogoutClick = () => {
        TokenService.clearAuthToken();
        window.location='/';
    }
    
    render() {
        return (
            <div className="Garden_Nav">
                <Link to="/">
                    Landing Page
                </Link>
                ||
                <Link to="/plant-rose">
                    Plant A Rose
                </Link>
                ||
                <Link to="/" onClick={this.handleLogoutClick}>
                    Sign Out
                </Link>
            </div>
        )
    }
}