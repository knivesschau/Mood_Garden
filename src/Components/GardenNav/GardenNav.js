import React, {Component} from 'react';
import {Link} from 'react-router-dom';
import './GardenNav.css'

export default class RoseNav extends Component {
    render() {
        return (
            <div className="Garden_Nav">
                <Link to="/">
                    Landing Page
                </Link>
                ||
                <Link to="/plant-rose">
                    Plant A Rose
                </Link>
                ||
                <Link to="/">
                    Sign Out
                </Link>
            </div>
        )
    }
}