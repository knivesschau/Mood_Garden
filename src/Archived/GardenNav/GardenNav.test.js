import React from 'react';
import {shallow} from 'enzyme';
import toJson from 'enzyme-to-json';
import GardenNav from './GardenNav';

describe.skip ('GardenNav Component', () => {
    it ('renders the complete GardenNav Component', () => {
        const wrapper = shallow(<GardenNav/>);
        expect(toJson(wrapper)).toMatchSnapshot();
    })
})

