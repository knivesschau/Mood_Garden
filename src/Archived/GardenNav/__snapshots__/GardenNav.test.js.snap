// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`GardenNav Component renders the complete GardenNav Component 1`] = `
<div
  className="Garden_Nav"
>
  <Link
    to="/"
  >
    Landing Page
  </Link>
  ||
  <Link
    to="/plant-rose"
  >
    Plant A Rose
  </Link>
  ||
  <Link
    onClick={[Function]}
    to="/"
  >
    Sign Out
  </Link>
</div>
`;
